index.js

```diff
@@ -27,11 +27,20 @@ const buildQueryParams = function(vin, scope, country, options) {
   if (options.flags) {
     parameters.flags = util.getFlagsString(options.flags);
   }
-
   if (options.hasOwnProperty('testMode')) {
-    parameters.mode = options.testMode ? 'test' : 'live';
+    // eslint-disable-next-line no-console
+    console.warn(
+      'testMode has deprecated, use mode to specify the mode instead',
+    );
+    parameters.mode = options.testMode === true ? 'test' : 'live';
+  } else if (options.hasOwnProperty('mode')) {
+    parameters.mode = options.mode;
+    if (!['test', 'live', 'simulated'].includes(parameters.mode)) {
+      throw new Error(
+        'mode MUST be one of the following \'test\', \'live\', \'simulated\'',
+      );
+    }
```

Comment:
Can you update the JSDoc for `getCompatibility` to document the `mode` parameter?